rcl/test/rcl/test_time.cpp

```diff
@@ -205,6 +205,29 @@ TEST_F(CLASSNAME(TestTimeFixture, RMW_IMPLEMENTATION), test_ros_clock_initially_
   EXPECT_EQ(0, query_now);
 }
 
+TEST(CLASSNAME(rcl_time, RMW_IMPLEMENTATION), time_point_validation) {
+  ASSERT_FALSE(rcl_time_point_valid(NULL));
+
+  rcl_time_point_t uninitialized;
+  (void) uninitialized;
+  // Not reliably detectable due to random values.
+  // ASSERT_FALSE(rcl_clock_valid(&uninitialized));
+
+  rcl_time_point_t valid, invalid;
+
+  valid.nanoseconds = 1000;
+  valid.clock_type = RCL_ROS_TIME;
```

Comment:
missing test case for `RCL_CLOCK_UNINITIALIZED`, so that we can detect that implementation is not completed?